detectors/node/opentelemetry-resource-detector-instana/README.md

```diff
@@ -35,15 +35,37 @@ const globalResource = new Resource({
 });
 
 const sdk = new NodeSDK({
-   autoDetectResources: false,
    resourceDetectors: [envDetector, processDetector, instanaAgentDetector],
    resource: globalResource,
 });
 
-(async () => {
-   await sdk.detectResources();
+sdk.start()
+```
+
+```typescript
+import {
+  Resource,
+  processDetector,
+  envDetector,
+} from "@opentelemetry/resources";
+import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";
+import { NodeSDK } from "@opentelemetry/sdk-node";
+import { instanaAgentDetector } from "@opentelemetry/resource-detector-instana";
 
-   await sdk.start();
+const globalResource = new Resource({
+   [SemanticResourceAttributes.SERVICE_NAME]: "TestService",
+});
+
+const sdk = new NodeSDK({
+   resourceDetectors: [envDetector, processDetector, instanaAgentDetector],
+   resource: globalResource,
+});
+
+sdk.start()
+
+(async () => {
+   const resource = sdk["_resource"];
```

Comment:
`_resource` is private therefore I don't think we should show this be in the readme.

If one has the need to explicit wait for the resource detection some API on `NodeSDK` should be added. But this should be no concern of all the detector readme files and handled outside of this PR.
Alternative is to await the resource detection before SDK creation.

Reply:
Ok removed this example.